src/main.py

```diff
@@ -54,14 +55,43 @@ def frontend_communication_upload_recorded():
         return jsonify({
             "error": "Expected file named recordingTemp"
         })
-    filename_ogg = app.config['UPLOAD_FOLDER'] / "recordingTemp.ogg"
+    filename_ogg = app.config['TEMP_FOLDER'] / "recordingTemp.ogg"
     file.save(filename_ogg)
-    filename_ogg = pathlib.Path(filename_ogg)
-    filename = app.config['UPLOAD_FOLDER'] / "recordingTemp.wav"
-    convert_recorded_file(filename_ogg, filename)
+    filename = app.config['TEMP_FOLDER'] / "recordingTemp.wav"
+    convert_recorded_to_wav(filename_ogg, filename)
     return jsonify({"filename": str(filename)})
 
 
+@app.route('/saveWithChords', methods=['POST'])
+def frontend_communication_save_with_chords():
+    try:
+        filename_dest = request.json["output_file"]
+    except Exception as e:
+        logger.error(f"Bad request: {request}\n Exception: {e}")
+        return jsonify({
+            "error": "Expected json with output_file key"
+        })
+    filename_src = app.config['TEMP_FOLDER'] / "output.midi"
+    music_synthesis.save_midifile_as_wav(filename_src, filename_dest)
+    return jsonify({})
+
+
+@app.route('/saveRecorded', methods=['POST'])
+def frontend_communication_save_recorded():
```

Comment:
What is the difference between /recorded and /saveRecorded ?

Reply:
You meand we should change the name to /recorded?

Reply:
/recorded saves file to temp, and /saveRecorded moves it to other location?

Reply:
Ah, yes. It is done this way so the user can record without choosing where to save the recording